### PR TITLE
Update changes

### PR DESCRIPTION
**Pull request by FrancoisLaferriere**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Changes
+
+## telingo-2.0.0
+  * add extension to handle dynamic formulas
```
Comment (tortinator) [inline CHANGES.md]: This is not true - please revise!

**Review by tortinator — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (rkaminsk) [reply CHANGES.md]: I thought this is true? Maybe it is not arbitrary dynamic formulas but such details we can describe in the readme. If it is more than this, please also have a look at the [README.md](https://github.com/potassco/telingo/blob/master/README.md#dynamic-logic).

**Comment by rkaminsk** [comment]
Let's stick with this pull request. :wink:

**Comment by rkaminsk** [comment]
The previous description might have been too general but we also do not have to be lawyers here nailing down every detail. That's for the documentation. What about `add limited support for dynamic formulas with path expressions`. I would probably not even write the `limited` part but also do not oppose it. :wink:

**Pull request merged**